e2e/src/tests/fileBrowserPage.test.js

```diff
@@ -0,0 +1,19 @@
+import { vBtn, vListItemSubtitle, vList } from 'jest-puppeteer-vuetify';
+import {
+  waitForRequestsToFinish,
+} from '../util';
+
+describe('file browser page', () => {
+  it('view file browser', async () => {
+    await expect(page).toClickXPath(vBtn('Public Dandisets'));
+    await waitForRequestsToFinish();
+
+    // Assert that the page has loaded with a list of dandisets
+    await expect(page).toClickXPath(vListItemSubtitle('DANDI:000001'));
```

Comment:
The assumption that dandiset 000001 is on the first page of the Public Dandisets page is only true for certain test execution orderings. In practice, it seems jest is picking it as one of the first tests it runs. But if it got scheduled after, say, the [search page test](https://github.com/dandi/dandi-archive/blob/master/e2e/src/tests/dandisetsPage.test.js#L31), it would end up on the second or third page at least.

One option could be to manually specify an order so that this test runs first (https://jestjs.io/docs/configuration#testsequencer-string), or just modify the code to navigate directly to `/dandisets/000001`/ instead of clicking through the UI to get there. Personally I think the second option sounds cleaner and also has the benefit of decreasing the scope of this test (since this test is only focused on the file browser, and doesn't need to concern itself with clicking through the Public Dandiset page)

Reply:
changed in 21e7b0f81fed2209d602daa987c6657bce2d1bf0